nbs/api/00_core.ipynb

```diff
@@ -3266,6 +3267,89 @@
     "assert '<title>Foo</title>' in txt and '<h1>bar</h1>' in txt and '<html>' in txt"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "94705ca7",
+   "metadata": {},
+   "source": [
+    "Test on a single background task:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8cf7433e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Starting slow task\n",
+      "Finished slow task\n"
+     ]
+    }
+   ],
+   "source": [
+    "def my_slow_task():\n",
+    "    print('Starting slow task')    \n",
+    "    time.sleep(3)\n",
```

Comment:
This will make the whole notebook run and test slowly. How about using 0.001 instead?

Reply:
I used to have it longer but made it shorter. Still shorter we will go!